tools/mason/MasonExternal.chpl

```diff
@@ -41,6 +41,8 @@ proc masonExternal(args: [] string) {
     else if args[2] == "--setup" {
       if isDir(SPACK_ROOT) then
         throw new owned MasonError("Spack backend is already installed");
+      else if MASON_OFFLINE == true then
+        throw new owned MasonError('Cannot setup Spack when MASON_OFFLINE is set to true');
```

Comment:
We need to disable the `Use --update to override` message for `mason external` errors.

```
> mason external install
Cannot install Spack packages when MASON_OFFLINE=true
Use --update to override
```